service/src/test/java/aero/minova/cas/service/SecurityServiceTests.java

```diff
@@ -52,10 +52,10 @@ void setUp() {
 	void test_rowLevelSecurityWithNoRoles() {
 		List<Row> userGroups = new ArrayList<>();
 
-		assertThat(securityService.rowLevelSecurity(false, userGroups))//
-				.isEqualTo("\r\nwhere ( ( SecurityToken IS NULL ) )");
-		assertThat(securityService.rowLevelSecurity(true, userGroups))//
+		assertThat(SecurityService.rowLevelSecurity(false, userGroups))//
```

Comment:
Warum hat sich hier in der Klasse die Groß-/Kleinschreibung geändert?
Die Variable oben ist aber nicht betroffen?

Reply:
Die Methode ist jetzt static. Der securityService wird aber an anderer Stelle in der Klasse noch benötigt.